values.yaml

```diff
@@ -1,76 +1,86 @@
+  #
+  # ```yaml
+  # # Consul 1.5.0
+  # image: "consul:1.5.0"
+  # # Consul Enterprise 1.5.0
+  # image: "hashicorp/consul-enterprise:1.5.0-ent"
+  # ```
   image: "hashicorp/consul:1.9.0"
 
-  # array of objects containing image pull secret names that will be applied to
-  # each service account.
-  # This can be used to reference image pull secrets if using
-  # a custom consul or consul-k8s Docker image.
-  # See https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry.
+  # Array of objects containing image pull secret names that will be applied to each service account.
+  # This can be used to reference image pull secrets if using a custom consul or consul-k8s Docker image.
+  # See https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry for reference.
   #
   # Example:
-  #   imagePullSecrets:
+  #
+  # ```yaml
+  # imagePullSecrets:
   #   - name: pull-secret-name
   #   - name: pull-secret-name-2
+  # ```
+  # @type: array<map>
   imagePullSecrets: []
 
-  # imageK8S is the name (and tag) of the consul-k8s Docker image that
-  # is used for functionality such as catalog sync and connect. This can be overridden
-  # per component.
+  # The name (and tag) of the consul-k8s (https://github.com/hashicorp/consul-k8s)
+  # Docker image that is used for functionality such the catalog sync.
+  # This can be overridden per component.
   imageK8S: "hashicorp/consul-k8s:0.21.0"
 
-  # datacenter is the name of the datacenter that the agents should register
-  # as. This can't be changed once the Consul cluster is up and running
-  # since Consul doesn't support an automatic way to change this value
-  # currently: https://github.com/hashicorp/consul/issues/1858.
+  # The name of the datacenter that the agents should
+  # register as. This can't be changed once the Consul cluster is up and running
+  # since Consul doesn't support an automatic way to change this value currently:
+  # https://github.com/hashicorp/consul/issues/1858.
   datacenter: dc1
 
-  # enablePodSecurityPolicies controls whether pod
-  # security policies are created for the Consul components created by this
-  # chart. See https://kubernetes.io/docs/concepts/policy/pod-security-policy/.
+  # Controls whether pod security policies are created for the Consul components
+  # created by this chart. See https://kubernetes.io/docs/concepts/policy/pod-security-policy/.
   enablePodSecurityPolicies: false
 
-  # gossipEncryption configures which Kubernetes secret to retrieve Consul's
-  # gossip encryption key from (see https://www.consul.io/docs/agent/options.html#_encrypt).
-  # If secretName or secretKey are not set, gossip encryption will not be enabled.
-  # The secret must be in the same namespace that Consul is installed into.
+  # Configures which Kubernetes secret to retrieve Consul's
+  # gossip encryption key from (see [-encrypt](/docs/agent/options#_encrypt)). If secretName or
```

Comment:
I'm curious why does the link for `-encrypt` look different from the one in `domain` i.e. it has `[]`?

Reply:
yeah this was a mistake. We need to make the urls absolute so folks just reading the yaml can use them.